__tests__/app.test.js

```diff
@@ -51,6 +51,37 @@ afterAll(() => {
         });
     });
 
+    describe("GET /api/articles/:article_id", () => {
+        test("200: should respond with an article object, matching the inputted article ID", () => {
+            return request(app)
+            .get("/api/articles/5")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body).toBeInstanceOf(Object);
+                expect(body.article[0].article_id).toBe(5);
```

Comment:
Article should be an object not an array if it's just singular

Reply:
also from previous ticket/PR